docs/cosmos-hub/genesis.md

```diff
@@ -0,0 +1,332 @@
+# Genesis File
```

Comment:
imo the SDK docs should keep everything on this file but the module specific genesis states